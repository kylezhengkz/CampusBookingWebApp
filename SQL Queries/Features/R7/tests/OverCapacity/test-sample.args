{
    "args": [
        "00000000-0000-0000-0000-000000000004",
        "00000000-0000-0000-0000-000000000003",
        "2028-01-17 10:30:05",
        "2028-01-17 11:30:19",
        9999
    ],
    "kwargs": {}
}   