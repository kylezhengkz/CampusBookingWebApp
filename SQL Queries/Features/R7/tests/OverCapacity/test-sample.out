RoomOverCapacityError: The number of participants in the booking (9999) is not smaller or equal to the capacity of the room (50)